README.md

```diff
@@ -1 +1,67 @@
-Tools for Wizzilab products
+# AppLink via python
+
+Python scripts to communicate with Wizzilab's AppLink. 
+
+- watch_reports.py allows to display reports coming from sensors
+- read_example.py allows to read a property of a device 
+- write_example.py shows how to configure a property of a device. 
```

Comment:
Why do the 2 first scripts `allow` some actions while the last `shows how to`.
It feels like the 2 first scripts are treated as tools whereas the last one is described as an example.
It thinks all should be treated as tools, and therefore the `write_example.py` should allow `to configure a property of a device`.